File: src/main/java/config/ConfigLoader.java
package config;

import utils.ArgsParser;

public class ConfigLoader {
    private AppConfig appConfig;

    public ConfigLoader(String[] arguments) {
        appConfig = new AppConfig(Settings.EDGE_DRIVER_PATH, new ArgsParser().parseArgs(arguments));
    }

    public AppConfig loadConfiguration() {
        return appConfig;
    }
}

File: config/ConfigLoader.py
from config.settings import EDGE_DRIVER_PATH
from utils.ArgsParser import ArgsParser


class ConfigLoader:
    @staticmethod
    def get_edge_driver_path(self):
        return EDGE_DRIVER_PATH

    @staticmethod
    def get_browser_options(self):
        self.detach = ArgsParser().parse_args()
        return self.detach
